dcs-packaging-tool/dcs-packaging-tool-gui/src/main/java/org/dataconservancy/packaging/gui/presenter/impl/BasePresenterImpl.java

```diff
@@ -46,8 +46,6 @@ protected void bindBaseElements() {
         view.getHeaderViewAboutLink().setOnAction(arg0 -> view.showAboutPopup());
         
         view.getHeaderViewHelpLink().setOnAction(arg0 -> view.showHelpPopup());
-        
-        view.getCancelLink().setOnAction(arg0 -> getController().goToPreviousPage());

```

Comment:
Sure but base Presenter is already abstract you could just define a method for the back behavior  in it and then override it in the presenters that require exception behavior like displaying a popup.